src/components/DownloadImageDialog/ImageForm.tsx

```diff
@@ -136,11 +136,14 @@ export const ImageForm = memo(function ImageForm({
 	const handleVersionChange = useCallback(
 		(ver: typeof version) => {
 			if (!ver) {
-				const newVersion =
+				ver =
 					versionSelectionOpts.find((v) => v.isRecommended) ??
 					versionSelectionOpts[0];
-				onChange('version', newVersion?.value);
```

Comment:
The early return here didn't allow the variant related code that we have later run :(